lib/stm32/f0/rcc.c

```diff
@@ -626,6 +709,34 @@ void rcc_clock_setup_in_hsi48_out_48mhz(void)
 	rcc_ahb_frequency = 48000000;
 }
 
+void rcc_clock_setup_pll(const struct rcc_clock_scale *clock)
+{
+	rcc_osc_on(clock->sysclk_source);
+	rcc_wait_for_osc_ready(clock->sysclk_source);
+	rcc_set_sysclk_source(clock->sysclk_source);
+
+	rcc_set_hpre(clock->hpre);
+	rcc_set_ppre(clock->ppre);
+
+	flash_prefetch_enable();
+	if (clock->ahb_frequency > 24000000)
+		flash_set_ws(FLASH_ACR_LATENCY_024_048MHZ);
```

Comment:
use proper { } brackets here.  You can run "make stylecheck" to verify.

Reply:
Logically though, is it really that simple on f0? it doesn't depend on voltage as well?  On other families, flash wait states are simply one of the parameters in the clock struct.

Reply:
Thx for the make stylecheck hint. Unfortunately overseen that. Added a coding style fix. 

 RM0360 (stm32f030x4x6x8xc reference manual) stated in 3.5.1
> Bits 2:0 LATENCY[2:0]: Latency
> These bits represent the ratio of the SYSCLK (system clock) period to the Flash access time.
> 000: Zero wait state, if SYSCLK ≤ 24 MHz
> 001: One wait state, if 24 MHz < SYSCLK ≤ 48 MH

Same part in RM0091 (stm32f0x1stm32f0x2stm32f0x8).
But rethought that part. A separate parameter seem to be the better and more defensive choice.
Reworked that.

Reordered the series and claim f50bef785391 ("stm32f0: rcc: Add support for 16MHz external clocks") as optional.
Unluckily I wrote that code in the first place... :)